docs/howto/deployment/asgi/uvicorn.txt

```diff
@@ -24,12 +24,30 @@ called (separated by a colon).
 
 For a typical Django project, invoking Uvicorn would look like::
 
-    gunicorn myproject.asgi:application -k uvicorn.workers.UvicornWorker
+    uvicorn myproject.asgi:application
```

Comment:
We can also suggest here to use `uvicorn` as a module?
```suggestion
    python -m uvicorn myproject.asgi:application
```

Reply:
Yup, this works! Thanks.